src/oas3/service.ts

```diff
@@ -38,6 +38,14 @@ export const transformOas3Service: Oas3HttpServiceTransformer = ({ document }) =
     httpService.termsOfService = document.info.termsOfService;
   }
 
+  if (document.info?.['x-logo']) {
+    httpService.logo = {
+      ...document.info['x-logo'],
+      altText: document.info['x-logo'].altText ?? 'logo',
+      href: document.info['x-logo'].href ?? document.info.contact?.url,
```

Comment:
This is a bit too naive.
it won't work correctly if a user provides a different value than a string, i.e. an array or a number.